packages/mds-types/device.ts

```diff
@@ -0,0 +1,39 @@
+import { Enum, Timestamp, UUID } from './utils'
+import { VEHICLE_STATE } from './vehicle/vehicle_states'
+import { VEHICLE_TYPE } from './vehicle/vehicle_types'
+
+export const PROPULSION_TYPES = Enum('human', 'electric', 'electric_assist', 'hybrid', 'combustion')
```

Comment:
Any reason to use `Enum` here rather than just a const array of string literals like the others? `Enum` seems to be a legacy thing from before string literals were a thing.